src/App.js

```diff
@@ -1,27 +1,49 @@
-import React from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React, { useEffect } from 'react';
+import {
+  BrowserRouter as Router,
+  Switch,
+  Route,
+  Link,
+  Redirect,
+} from 'react-router-dom';
+import { fetchQuestions } from 'actions/';
+import { useDispatch, useSelector } from 'react-redux';
+import Quiz from 'components/quiz/';
+import Results from 'components/results/';
+import 'App.scss';
+
+const App = () => {
+  const auth = useSelector(state => state.auth);
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(fetchQuestions());
+  }, [dispatch]);
 
-function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          <code>src/App.js</code>
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer">
-          Learn React
-        </a>
-      </header>
+    <div className="app">
+      <div className="app__title">
+        <h1> QUIZ APP </h1>
+      </div>
+      <Router>
+        <Switch>
+          <Route exact path="/">
+            <div className="container app__welcome">
+              <h2 className="app__heading app__heading--big">
+                Welcome to quiz app
+              </h2>
+              <Link to="/ex" className="app__link">
+                Start
+              </Link>
+            </div>
+          </Route>
```

Comment:
`<Route exact path="/">
            <div className="container app__welcome">
              <h2 className="app__heading app__heading--big">
                Welcome to quiz app
              </h2>
              <Link to="/ex" className="app__link">
                Start
              </Link>
            </div>
          </Route>`
nội dung trong <Route /> không nên viết dài dòng thế này, chỉ cần sử dụng exact, path, render.